README.md

```diff
@@ -161,7 +161,8 @@ Client Instrumentation|Every Tuesday at 9:00 AM PT (primary) and every Monday at
 Kubernetes Operator|Every other week on Thursday at 09:00 PT|[Google Doc](https://docs.google.com/document/d/1Unbs2qp_j5kp8FfL_lRH-ld7i5EOQpsq0I4djkOOSL4)|[Slack](https://cloud-native.slack.com/archives/C033BJ8BASU)|
 Community Demo Application|Every Monday at 8:15 PT|[Google Doc](https://docs.google.com/document/d/16f-JOjKzLgWxULRxY8TmpM_FjlI1sthvKurnqFz9x98/edit)|[Slack](https://cloud-native.slack.com/archives/C03B4CWV4DA)|
 Functions as a Service (FAAS) |Every Tuesday at 12:00 PM PT |[Google Doc](https://docs.google.com/document/d/187XYoQcXQ9JxS_5v2wvZ0NEysaJ02xoOYNXj08pT0zc/edit#)|[Slack](https://cloud-native.slack.com/archives/C04HVBETC9Z)|
-OpenTelemetry Profilng WG | Every other week on Thursday at 08:00 AM PT |[Google Doc](https://docs.google.com/document/d/19UqPPPlGE83N37MhS93uRlxsP1_wGxQ33Qv6CDHaEp0/edit?usp=sharing)|[Slack](https://cloud-native.slack.com/archives/C03J794L0BV)|
+OpenTelemetry Profiling WG | Every other week on Thursday at 08:00 AM PT |[Google Doc](https://docs.google.com/document/d/19UqPPPlGE83N37MhS93uRlxsP1_wGxQ33Qv6CDHaEp0/edit?usp=sharing)|[Slack](https://cloud-native.slack.com/archives/C03J794L0BV)|
+Security | Weekly on Wednesday at 08:30 AM PT |[Google Doc](https://docs.google.com/document/d/1P2xejC7lEkOV_Z-8E0oZPXLK5HOnUPNuRqKP0ZQ5fpg/edit?usp=sharing)|[Slack](https://cloud-native.slack.com/archives/C05A85QC281)|
```

Comment:
maybe, to avoid the confusion that this is a related to something for end users?
```suggestion
Security Governance | Weekly on Wednesday at 08:30 AM PT |[Google Doc](https://docs.google.com/document/d/1P2xejC7lEkOV_Z-8E0oZPXLK5HOnUPNuRqKP0ZQ5fpg/edit?usp=sharing)|[Slack](https://cloud-native.slack.com/archives/C05A85QC281)|
```

Reply:
There was a discussion about this at the call today. A few different names were discussed, but the group present settled on "Security SIG" to keep it simple and it also aligns w/ the [Kubernetes Security SIG](https://github.com/kubernetes/sig-security#security-special-interest-group), which fulfills the same role